app/src/main/res/values-night/colors.xml

```diff
@@ -16,6 +16,8 @@
     <color name="title_text_next_to_image">#fff</color>
     <color name="description_text_next_to_image">#fff</color>
 
+    <color name="smoothness_vehicle_tyre">#70808b</color>
```

Comment:
whoa, ... man, okay...

Why not simply make the tyre color darker than the dark mode background?

Reply:
I tried that first, but I couldn't find a color that looks nice on both bright and dark backgrounds.